### ISSUE TITLE
Allow underscore values in the URL path part

### ISSUE THREAD

**Issue by arzljames** [zesty-io/manager-ui#2577]
**Is your feature request related to a problem? Please describe.**
Some customers have requested to change their URL path parts to include underscores as values. While this can be updated through the API, it would be nice if the users themselves had the capacity to update it in the UI.

e.g `mywebsite.com/about-page` to `mywebsite.com/about_page`
![image](https://github.com/zesty-io/manager-ui/assets/70579069/0d2c901e-7b5e-4220-b228-71f3cbaf6785)

**Comment by zcolah** [comment]
@arzljames I'd be glad to add this ticket to the roadmap but do want to bring up that it is not recommended for SEO to include underscores. I learned about this from @shrunyan when I joined Zesty. 

You can read more on this here: 

1. https://www.woorank.com/en/blog/underscores-in-urls-why-are-they-not-recommended
2. https://www.semrush.com/blog/the-most-common-hreflang-mistakes-infographic/
3. https://www.semrush.com/kb/542-site-audit-issues-list#underscore-url

I would recommend sharing this with the customer if @giseleblair approves.

**Comment by arzljames** [comment]
> @arzljames I'd be glad to add this ticket to the roadmap but do want to bring up that it is not recommended for SEO to include underscores. I learned about this from @shrunyan when I joined Zesty.
> 
> You can read more on this here:
> 
> 1. https://www.woorank.com/en/blog/underscores-in-urls-why-are-they-not-recommended
> 2. https://www.semrush.com/blog/the-most-common-hreflang-mistakes-infographic/
> 3. https://www.semrush.com/kb/542-site-audit-issues-list#underscore-url
> 
> I would recommend sharing this with the customer if @giseleblair approves.

Got it. This is insightful. Thanks.

**Comment by giseleblair** [comment]
@zcolah I agree with SEO best practices, but the API allows for it and so does Webengine. This was a request that came directly from the customer so we just want to match the UI to our API service in this scenario. If you'd like to scope in a warning message about this in the UI then that's definitely an option
